Gruntfile.js

```diff
@@ -85,8 +85,51 @@ module.exports = function(grunt) {
             }
         },
         clean: {
+            coverage: ['coverage/', 'coverage.lcov', '.nyc_output/'],
             dist: ['.dist/'],
+            packages: ['Consus-Client-*/'],
             test: ['.test/']
+        },
+        electron: {
+            win32Build: {
```

Comment:
This should be changed to `win64Build`, since this rule is actually for the x64 architecture (or add packages for 32-bit platforms?).